ts/pubsub.js

```diff
@@ -40,8 +36,8 @@ class BasicPubSub extends Pubsub {
     super({
       debugName,
       multicodecs,
-      peerId,
-      registrar,
+      peerId: libp2p.peerId,
+      registrar: libp2p.registrar,
```

Comment:
When we refactor pubsub, I think we should also send libp2p to it. What do you think?

Reply:
for sure :+1: